### DISCUSSION TITLE
ERROR: database "prisma_migrate_shadow_db_698acb48-e2a9-4be8-b5cc-97b5ad1a16e8" is being accessed by other users DETAIL: There is 1 other session using the database

### DISCUSSION THREAD

**Discussion by Susanta-0007** [supabase/supabase#20825] [Questions]
![Screenshot 2024-01-28 225232](https://github.com/supabase/supabase/assets/89796600/bb1f72d4-bdde-49e1-9483-acd4b622ab8f)
DETAIL: There is 1 other session using the database.
   0: schema_core::state::DevDiagnostic
             at schema-engine\core\src\state.rs:267

**Comment by GaryAustin1** [comment]
The main thread on the migration to supavisor.
https://github.com/orgs/supabase/discussions/17817

Also make sure to check the latest Prisma guide: https://supabase.com/partners/integrations/prisma
